scripts/clone_helix_lsp_config.py

```diff
@@ -11,7 +11,8 @@
 
 for lsp in lsps.values():
     # Margs command and args
-    lsp["cmd"] = [lsp["command"], *lsp.get("args", [])]
+    cmd = [lsp["command"], *lsp.get("args", [])]
+    lsp["cmd"] = cmd
```

Comment:
I don't think there's any need for this: `cmd` is not actually reused anywhere, so this change is just adding a variable assignment unnecessarily.